eslint/react/.eslintrc

```diff
@@ -18,13 +18,64 @@
 	},
 	"rules": {
 		"class-methods-use-this": "off",
+		"jsx-quotes": "error",
```

Comment:
is it for forcing single or double quotes? in my case i prefer double quotes, as is more html flavoured

Reply:
prefer-double is the default

Reply:
by default is double